packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -478,20 +481,20 @@ class ChartController extends DisposableController
     return traceIndex;
   }
 
-  int get numberOfVisibleXAxisTicks => visibleTicks;
+  int? get numberOfVisibleXAxisTicks => visibleTicks;
```

Comment:
let's remove this getter and just use visibleTicks instead wherever numberOfVisibleXAxisTicks was used. If we need to rename visibleTicks to visibleXAxisTicks to make this more clear, let's do that.

Reply:
done